src/custom/components/CowBalanceButton/index.tsx

```diff
@@ -76,22 +86,25 @@ interface CowBalanceButtonProps {
   account?: string | null | undefined
   chainId: ChainId | undefined
   onClick?: () => void
+  isUpToSmall?: boolean
 }
 
 const COW_DECIMALS = COW[ChainId.MAINNET].decimals
 
-export default function CowBalanceButton({ onClick }: CowBalanceButtonProps) {
+export default function CowBalanceButton({ onClick, isUpToSmall }: CowBalanceButtonProps) {
   const { balance, isLoading } = useCombinedBalance()
 
-  const formattedBalance = formatSmartLocaleAware(balance, AMOUNT_PRECISION)
+  const formattedBalance = formatSmartLocaleAware(balance, 0)
   const formattedMaxBalance = formatMax(balance, COW_DECIMALS)
 
   return (
     <Wrapper isLoading={isLoading} onClick={onClick}>
       <CowProtocolLogo />
-      <b title={formattedMaxBalance && `${formattedMaxBalance} (v)COW`}>
-        <Trans>{formattedBalance || 0}</Trans>
-      </b>
+      {!isUpToSmall && (
+        <b title={formattedMaxBalance && `${formattedMaxBalance} (v)COW`}>
```

Comment:
Is this still vCOW? Or does the `(v)` means it could be either?
![Screen Shot 2022-05-31 at 10 21 28](https://user-images.githubusercontent.com/43217/171140263-f4076418-d8d7-40b0-8f1a-afc2fa4ac1a3.png)

I'm honestly not sure which is the case

Reply:
(v)COW was meant to signal both your vCOW and COW balance in a single aggregate balance number. I haven't changed the logic here otherwise (just the precision was set to 0).